tests/lib/edge_impulse/src/main.cpp

```diff
@@ -141,6 +145,10 @@ static void result_ready_cb(int err)
 
 static void *test_init(void)
 {
+	static bool init_once;
+	if(init_once) return NULL;
+	init_once = true;
```

Comment:
Why do we need that? Isn't test_init called once?

``` suggestion
	static bool init_once;

	if(init_once) {
		return NULL;
	}

	init_once = true;
```

Reply:
yes, double init of this part is not allowed